website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -19,10 +19,18 @@ Use the [Account API](/cloud-docs/api-docs/account) to get account details, upda
 
 ## Creating an Account
 
-Creating an account requires a username, an email address, and a password. To use Terraform Cloud, you must create an account through one of the following methods:
+The recommended method to access Terraform Cloud is with [your HCP account](#continue-with-hcp-account). Alternatively, you may create an account through one of the following methods:
 
 - **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you will automatically join that organization and can begin using Terraform Cloud.
-- **Sign-Up Page:** For Terraform Cloud, go to [`https://app.terraform.io/signup/account`](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
+- **Sign-Up Page:** Creating an account requires a username, an email address, and a password. For Terraform Cloud, go to [`https://app.terraform.io/public/signup/account`](https://app.terraform.io/public/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/public/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
+
+### Continue with HCP Account
+
+Accessing Terraform Cloud with [your HCP account](https://cloud.hashicorp.com/docs/hcp/create-account) can be done by clicking **Continue with HCP account** button from the **Sign-In** page. 
+
+In the first sign-in, a new Terraform Cloud account will be created and linked to [your HCP account](https://cloud.hashicorp.com/docs/hcp/create-account).
+
+If there is an existing and still unlinked Terraform Cloud account with the same email address from [your HCP account](https://cloud.hashicorp.com/docs/hcp/create-account), you'll be able to link those accounts. After this process you won't be able to sign-in using Terraform Cloud's username and password, though you'll be able to unlink your account in **User Settings** page if you choose to.
```

Comment:
Do we want to add a sentence that some TFC features will be depreciated (MFA) and managed by HCP?

Reply:
I didn't add it on purpose. That's because there's detailted instructions on how to proceed in TFC's settings page for each one of those features. I feel like the more we add/repeat ourselves here the harder it will be to keep everything up-to-date.

To me the main intention of this change is to warn the user that HCP is a thing (and recommended) and that he won't be able to sign-in with TFC's username/password. I'm not 100% sure about this, that's why I've reinforced feedback request in our channel 😄